src/components/Modal/Modal.scss

```diff
@@ -102,4 +121,5 @@ $block: '.#{variables.$ns}modal';
 :root {
     --yc-modal-border-radius: 5px;
     --yc-modal-margin: 20px;
+    --yc-modal-with-inner-scroll-margin: 24px;
```

Comment:
From v6 we introduce public CSS API, this looks like a private. Please use SCSS var or name it like `--_--inner-scroll-margin` on component's root, not `:root`